tailwind.config.js

```diff
@@ -148,6 +148,138 @@ module.exports = {
   plugins: [
     require('@tailwindcss/typography'),
     require('@tailwindcss/container-queries'),
+    // https://github.com/tailwindlabs/tailwindcss/issues/5989#issuecomment-962048436
+    function ({ addUtilities, addComponents, addBase, theme }) {
+      addBase({
```

Comment:
Sometimes there was an error in the console, when you were running the project, that some class does not exists and to add it using the `@layer` directive and here's the explanation from the [issue that I've pasted in line 151](https://github.com/tailwindlabs/tailwindcss/issues/5989#issuecomment-962048436):
```
Again, what is happening is that your build tool is effectively running all of these processes in parallel:

tailwindcss -i global.css
tailwindcss -i module-one.module.css
tailwindcss -i module-two.module.css
tailwindcss -i module-three.module.css
When you realize this it is a bit less surprising that code in module-one.module.css can't "see" code in global.css. But if they share a config file and the code is in the config file, you are all good.
```
By moving everything that was using this `@layer` directive from `main.global.css` to `tailwind.config.js` I have resolved the problem described above. Because of this issue, I wasn't able to run Storybook and it would be hard to test Datepicker component, because it's not used anywhere in the app right now (in this PR).